### DISCUSSION TITLE
Is it possible to organize a server cluster using a subscription?

### DISCUSSION THREAD

**Discussion by denncom** [Gozargah/Marzban#238] [Q&A]
Is it possible to use a single subscription link to specify that multiple servers from different locations are added to the user?

**Comment by SaintShit** [comment]
It's going to be possible using [Marzban-node](https://github.com/Gozargah/Marzban-node). this is currently on beta version of marzban and has not been merged with latest version yet.

**Reply by denncom** [reply]
Wow, cool! I want to test this feature. Can you tell me how to install and test it?
